Makefile

```diff
@@ -298,16 +298,17 @@ endif
 
 cri-release: releases/$(CRIRELEASE).tar.gz
 	@echo "$(WHALE) $@"
-	@cd releases && sha256sum $(CRIRELEASE).tar.gz >$(CRIRELEASE).tar.gz.sha256sum
+	@cd releases && sha256sum $(CRIRELEASE).tar.gz >$(CRIRELEASE).tar.gz.sha256sum && ln -sf $(CRIRELEASE).tar.gz cri-containerd.tar.gz
```

Comment:
the links are needed by existing scripts i presume?

Reply:
assuming those scripts do not care about the SHA file

Reply:
Right now just uses the symlink to resolve the latest, but I think it makes more sense to sha256 check the actual file rather than the link.